start_scripts/README.md

```diff
@@ -3,12 +3,12 @@
 ## start_all_2022.sh
 
 Start script adapted from Self Racing Cars 2022, to be used with the Adafruit GPS module
-Make sure that the GPS is on /dev/ttyUSB0
-Make sure that the Arduino is on /dev/ttyACM0
+Make sure that the GPS is on /dev/gps
+Make sure that the Arduino is on /dev/arduino
```

Comment:
We can delete this in fact, the script does not exist anymore

Reply:
Done